app/top-page.module.css

```
@@ -0,0 +1,362 @@
+.footer {
+  margin-top: 60px;
+  background:linear-gradient(150deg, #0b69eb 70%, #ffffff 100%);
+.footer {
+  margin-top: 60px;
+  background:linear-gradient(150deg, `#0b69eb` 70%, `#ffffff` 100%);
+  text-align: center;
+  color: rgb(255, 255, 255);
+  padding:50px;
+  font-size: 1.2rem;
+}
+  text-align: center;
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)